perfzero/lib/cloud_manager.py

```diff
@@ -286,6 +297,17 @@ def parse_arguments(argv, command):  # pylint: disable=redefined-outer-name
         type=str,
         help='Zone of the instance to create.'
         )
+    parser.add_argument(
+        '--ssh-internal-ip',
+        action='store_true',
```

Comment:
Since the default value is set to True, can you explain when we will need to set it false? I am wondering if the addition of this parameter will disrupt existing usage of cloud_manager.py.

Reply:
The default value is false.  It will be set to true if user specifies this flag.

https://docs.python.org/3/howto/argparse.html#introducing-optional-arguments

Reply:
Yeah you are right. I somehow misunderstood `store_true` as meaning default value = True.

LGTM.